hudi-common/src/main/java/org/apache/hudi/common/model/HoodieWriteStat.java

```diff
@@ -290,7 +291,7 @@ public long getTotalRollbackBlocks() {
     return totalRollbackBlocks;
   }
 
-  public void setTotalRollbackBlocks(Long totalRollbackBlocks) {
```

Comment:
same here, we can address this as part of some other refactoring diff

Reply:
Discussed offline. Without this we are not able to read certain archived commits